desc/objectives/objective_funs.py

```diff
@@ -516,6 +513,12 @@ def _set_derivatives(self, use_jit=True):
 
     def _check_dimensions(self):
         """Check that len(target) = len(weight) = dim_f."""
+        if len(self.target) == 0:
+            # need this so if the objective was previously built at a resolution where
+            # target is [] and the resolution changes the target and weight get
+            # set correctly
+            self._target = np.zeros(1)
+            self._weight = np.ones(1)
```

Comment:
It was with the ``FixLambdaGauge`` constraint. If the initial equilibrium is both stellarator symmetric and has no toroidal resolution, then ``dim_f`` for that objective is zero. Then when we increase toroidal resolution we need the target and weight arrays to have at least size 1 so that they can be broadcasted properly